.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
     strategy:
       matrix:
         rails_version: [5.2.3, 6.1.1, main]
-        ruby_version: [2.5.x, 2.6.x, 2.7.x, 3.0.x]
+        ruby_version: [2.6.x, 2.7.x, 3.0.x]
```

Comment:
what do these builds do?

Reply:
What do you mean?

Reply:
Also, what would you think about adding 3.1.x?

Reply:
what do these benchmark checks do? just curious

Reply:
Ohhh I see. They don't really do anything as far as I know 😅 I think we have them in case we want to look back at the numbers to see when regressions were introduced. We should probably be comparing them to some baseline tho.